ormar/actions.py

```diff
@@ -0,0 +1,26 @@
+"""
+Gathers all referential actions by ormar.
+"""
+
+
+from enum import Enum
+
+
+class Action(Enum):
+    """
+    Because the database management system(DBMS) enforces referential constraints,
+    it must ensure data integrity
+    if rows in a referenced table are to be deleted (or updated).
+
+    If dependent rows in referencing tables still exist,
+    those references have to be considered.
+
+    SQL specifies 5 different referential actions
+    that shall take place in such occurrences.
+    """
+
+    CASCADE: str = "CASCADE"
+    RESTRICT: str = "RESTRICT"
+    SET_NULL: str = "SET NULL"
+    SET_DEFAULT: str = "SET DEFAULT"
```

Comment:
I would leave that despite you are not really allowed to set default on fk in ormar, you can still do it in database directly

Reply:
My question is why we are not allowed to set `default` in ormar?
If we add this possibility, what problem might occur?

Reply:
Because the default values in pydantic are assigned in init, but init cannot be async.
You cannot save anything to the database in init (bad practice anyway). 
That means that the default value set on the foreign key would have to already exist in db prior to model declaration, it won't be created with the parent model. 

That would be super confusing for users that would want to add a whole related model as default in the parent model declaration (wouldn't work). And would require creating and **saving** the default model in db before the model that uses the it as default value.

Reply:
Excellent, I completely understand
So, I will add a note in the document that to use `SET_DEFAULT`, the `server_default` must be specified.